tests/unit/Model/Response.model.test.js

```diff
@@ -6,6 +6,7 @@ describe('Model/ResponseModel', () => {
     expect(response.generate().headers).toEqual(RESPONSE_HEADERS);
   });
 
+
```

Comment:
Stray blank line; usually eslint picks these up.

Reply:
Weird, I saved the file again and it removed it...
3c89a6fdb05551d2de7447cfa4b8496ac2999011